example_dags/example_transform.py

```diff
@@ -28,12 +28,12 @@ def top_five_animations(input_table: Table):
 
 # [START transform_example_2]
 @aql.transform()
-def last_five_animations(input_table: Table):
+def last_five_animations(input_table: Table):  # skipcq: PYL-W0613
```

Comment:
`# skipcq: PYL-W0613` can be removed. I feel @kaxil  has added it globally.

Reply:
@sunank200 mentioned before, this check was failing 3h ago in this PR :(